### ISSUE TITLE
[question] [Conan2.0] change of behaviour in `conan --version`

### ISSUE THREAD

**Issue by foundry-markf** [conan-io/conan#10015]
Hi,

I have just started trying the Conan 2.0.0a1 package out, and noted a change of behaviour for the `conan --version` command. Noticed this because I have existing scripts to get this information that work in 1.x.

With 1.x, if I do the following, the version string is on stdout
```
(env38) D:\dev\SharedTech\vfx20\recipes>conan --version >out.txt 2>err.txt

(env38) D:\dev\SharedTech\vfx20\recipes>type out.txt
Conan version 1.42.1

(env38) D:\dev\SharedTech\vfx20\recipes>type err.txt

(env38) D:\dev\SharedTech\vfx20\recipes>
```

With 2.0.0a1, it's now on stderr:
```
(env38) D:\dev\SharedTech\vfx20\recipes>conan --version >out.txt 2>err.txt

(env38) D:\dev\SharedTech\vfx20\recipes>type out.txt

(env38) D:\dev\SharedTech\vfx20\recipes>type err.txt
Conan version 2.0.0-alpha1
```

Not a biggy, but might catch others out, if this was intentional?

- [X] I've read the [CONTRIBUTING guide](https://github.com/conan-io/conan/blob/develop/.github/CONTRIBUTING.md).

**Comment by memsharded** [comment]
Hi @foundry-markf 

The idea is that the main output of commands goes to stdout, while all the "informational" stuff goes to stderr. In this case, I'd say the version information is exactly the expected output of the command, so it should go to stdout. Lets fix it, thanks for reporting.

**Comment by franramirez688** [comment]
Thanks for reporting the issue @foundry-markf 
This change will be released in the next alfa version `2.0.0-alpha2`.
